src/detections/table/mobile/body/row.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { TableCell, TableRow } from '@material-ui/core'
+
+import { useNavigation } from 'detections/table/hooks/use-navigation'
+import { useRowsStyle } from 'detections/table/hooks/use-rows-style'
+
+export const Row = ({ item }) => <TableRow onClick={useNavigation(`/detection/${item.id}`)} classes={useRowsStyle()}>
+  <TableCell align={'center'}>{item.class}</TableCell>
```

Comment:
For mobile, let's display the `class` and `timestamp` properties, as they are probably the most relevant ones.

Reply:
Also, instead of reimplementing our entire table just to hide a couple of cells, wouldn't it be better to use a [media query hook](https://material-ui.com/components/use-media-query/) and conditionally add the `hidden` prop on the required table cells?

Reply:
Done

Reply:
I was actually referring to something more like this:
![image](https://user-images.githubusercontent.com/23569938/129038069-f72fa34c-8361-402c-88e2-06e062e880e6.png)

We can transform our array from an array of strings to an array of objects, like this: 
![image](https://user-images.githubusercontent.com/23569938/129038154-9eb7dd53-b75d-4f47-b61b-388b8dd3a3b2.png)

Thus, we only implement each component once. All the display logic is handled by hooks.

Reply:
Done again